### DISCUSSION TITLE
Support Nextion 3.5 inch

### DISCUSSION THREAD

**Discussion by jvle** [Zer0-bit/gaggiuino#330] [Q&A] [answered]
Can I use the existing nexton-lcd.HMI, change device to 3.5 nextion display, the move and resize some buttons around (without changing nay opcode sent to Arduino? Would that work without changing Arduino source code?

**Comment by Zer0-bit** [comment, marked as answer]
Should work if you know what you're doing.

**Comment by EngMarc** [comment]
I've ordered both the the smaller TFT and the 3.5 TFT. I'm planning to scale up and significantly modify the UI so wanted to try both sizes.

**Comment by McTecnic** [comment]
If in the nextion LCD Editor to the corresponding device, e.g. 3.5", at least the 2.4" surface in the upper left corner of the 3.5" LCD runs in the debugger.
After adjusting the position of the buttons and graphics, the simulation also works.

**Reply by EngMarc** [reply]
Great, nice to know and thanks.

**Comment by kohlwebb** [comment]
How did this go? Were you able to successfully size up the interface?
